src/Sylius/Bundle/CoreBundle/Form/Type/Product/ChannelPricingType.php

```diff
@@ -48,6 +62,15 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 return;
             }
 
+            /** @var ChannelInterface $channel */
+            $channel = $options['channel'];
+
+            if (!$channel->isEnabled() && $channelPricing->getPrice() === null) {
```

Comment:
I'm wondering if we should cover the original price as well in this condition

Reply:
This is next bug, I will open new PR with next scenario to cover it